PWGHF/D2H/Utils/utilsRedDataFormat.h

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file utilsEvSelHf.h
+/// \brief Event selection utilities for reduced data format analyses
+/// \author Luca Aglietta <luca.aglietta@cern.ch>, UniTO Turin
+
+#ifndef PWGHF_D2H_UTILS_UTILSREDDATAFORMAT_H_
+#define PWGHF_D2H_UTILS_UTILSREDDATAFORMAT_H_
+
+#include "PWGHF/Core/CentralityEstimation.h"
+#include "PWGHF/Utils/utilsEvSelHf.h"
+
+namespace o2::hf_evsel
+{
+HfEventSelection hfEvSel;
```

Comment:
This should not be defined here, it should rather be an argument of the `checkEvSel` method

Reply:
Thank you for your comment, I missed it. I fixed it in the current version!